src/control/content/containers/Home.js

```diff
@@ -1,33 +1,254 @@
+          imageList = result.data.images;
+          if (!imageList.length) {
+            seeder.requestResult.complete();
+            return buildfire.dialog.toast({
+              message: "Bad AI request, please try changing your request.",
+              type: "danger",
+            });
+          }
+
+            const promises = imageList.map(
+              item => 
+              {
+                return new Promise(resolve => {
+                  const image = new Image();
+                  checkImageUrl(item.url).then(isValid => {
+                    image.onload = () => resolve(image);
+                      // adding cache buster to fix image duplication
+                      image.src = `${item.url}?v=${generateRandomNumber()}`;
+                      image.originalSrc = item.url + (isValid ? '' : '?valid=image-not-valid');    
+                  })
+                })
+              }
+            );
+            if (!promises.length){
+              seeder.requestResult.complete();
+              return buildfire.dialog.toast({
+                message: "Bad AI request, please try changing your request.",
+                type: "danger",
+              });
+            }
+
+            Promise.all(promises).then(items => {
+              let processedImages = items.map(
+                ({ naturalWidth, naturalHeight, originalSrc }) => {
+                  return  new Img({
+                      src: originalSrc,
+                      width: naturalWidth,
+                      height: naturalHeight,
+                      isValid: !originalSrc.includes('image-not-valid')
+                    })
+                }
+              );
+
+              processedImages = processedImages.filter(el => el.isImageValid);
+              if (!processedImages.length) {
+                seeder.requestResult.complete();
+                return buildfire.dialog.toast({
+                  message: "Bad AI request, please try changing your request.",
+                  type: "danger",
+                });
+              }
+
+              const processedFolders = [
+                new Folder({ name: result.data.topic, images: processedImages })
+              ];
+              const data = {
+                images: processedImages,
+                folders: processedFolders,
+              };
+  
+              saveSampleAiData(data, 'generate');
+              seeder.requestResult.complete();
+            });
+        },
+      },
+      importOptions: {
+        jsonTemplate: {
+          folders: [{ name: '', images: [{ url: '' }] }]
+        },
+        sampleCSV: 'Weather: https://source.unsplash.com/featured/?sunny\n'
+          + 'Weather:https://source.unsplash.com/featured/?rainy\n'
+          + 'City:https://source.unsplash.com/featured/?street',
+        systemMessage: 'Each row contains a folder and image URL that should belong to. if nothing provided or no URLs provided then return null',
+        maxRecords: 10,
```

Comment:
make it 25

Reply:
I've made it 5 after testing, it's the most stable choice